plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/handler/NewApplicationHandler.java

```diff
@@ -32,6 +32,10 @@
 	public Object execute(ExecutionEvent event) throws ExecutionException {
 		try {
 			WizardUtils.openWizardDialog(new NewApplicationWizard(), HandlerUtil.getActiveShell(event));
+
+			// This technically sends the wrong event for the wrong 'command' id,
+			// but the refresh resource handler only looks at the selection... 
+			new RefreshResourceHandler().execute(event);  
```

Comment:
I believe the proper fix for refresh will be available from the new openshift explorer layout feature branch, so this hack should be removed